userspace/libsinsp/mesos_http.h

```diff
@@ -218,6 +224,7 @@ class mesos_http
 	typedef std::shared_ptr<Json::Value> json_ptr_t;
 	static json_ptr_t try_parse(const std::string& json)
 	{
+		errstr = "";
```

Comment:
in this part of the file there are some errors (try it on OS X for example).
`errstr`, `g_json_error_log` and `g_logger` are not defined.

Reply:
Removed. That path is only used when reading trace files, I believe.